src/views/common/Login.vue

```diff
@@ -1,77 +1,79 @@
                             id="email"
                             v-model="email"
+                            autofocus
                             type="text"
-                            :class="[hasError ? 'border-red-500' : 'border-gray-300']"
-                            class="appearance-none rounded-lg block w-full px-3 py-3 border-2 placeholder-gray-500 text-gray-900 focus:outline-none focus:shadow-outline-blue focus:border-blue-300 sm:text-sm sm:leading-5"
-                            placeholder=""
+                            :class="[hasError ? 'input-text-error-tmp' : 'input-text-tmp']"
+                            class="w-full bg-white"
                             @input="resetError"
                         />
                     </div>
                     <div class="mt-6 relative">
-                        <label class="block mb-1 text-gray-700 font-medium tracking-wide">Password</label>
+                        <label class="input-label-tmp">Password</label>
                         <i
                             :class="[isPasswordVisible() ? 'fa-eye-slash' : 'fa-eye']"
-                            class="fas absolute z-20 mt-4 mr-4 right-0 text-gray-500 cursor-pointer"
+                            class="fas absolute z-20 mt-2.5 mr-4 right-0 text-gray-500 cursor-pointer"
                             @click="togglePassword"
                         />
                         <input
                             id="password"
                             v-model="password"
                             :type="passwordFieldType"
-                            :class="[hasError ? 'border-red-500' : 'border-gray-300']"
-                            class="appearance-none rounded-lg block w-full px-3 py-3 pr-12 border-2 placeholder-gray-500 text-gray-900 focus:outline-none focus:shadow-outline-blue focus:border-blue-300 sm:text-sm sm:leading-5"
-                            placeholder=""
+                            :class="[hasError ? 'input-text-error-tmp' : 'input-text-tmp']"
+                            class="w-full bg-white"
                             @input="resetError"
                         />
                     </div>
                 </div>
 
-                <div class="mt-10">
-                    <button
-                        id="login"
-                        :disabled="isInputEmpty"
-                        class="group relative w-full flex justify-center py-3 px-4 border border-transparent text-sm leading-5 font-medium rounded-md text-white bg-blue-600 hover:bg-blue-700 focus:outline-none focus:border-indigo-700 focus:shadow-outline-indigo active:bg-indigo-700 transition duration-150 ease-in-out disabled:bg-blue-300 disabled:cursor-not-allowed"
-                        @click="login"
-                    >
-                        Login
-                    </button>
+                <div class="mt-8">
+                    <button id="login" class="w-full btn-add-tmp" @click="login">Login</button>
```

Comment:
Button should be disabled when no or partial inputs are given

Reply:
It's a matter of taste, it is common practice to not deactivate the login button (see Instagram for example)